### PR TITLE
Fixes #927

### LINKED ISSUE #927: [🐛 Bug]: Outdated documentation: "Fresh browser per test"
### What happened?

The documentation at https://www.selenium.dev/documentation/test_practices/encouraged/fresh_browser_per_test/ is recommending to start a new `FirefoxDriver` with a certain `FirefoxProfile`.

If I understand the API correct, Selenium 4.1.1 does not support creating a new `FirefoxDriver` with a `FirefoxProfile` as parameter.
See [Selenium Java API for FirefoxDriver](https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/firefox/FirefoxDriver.html).

### How can we reproduce the issue?

```shell
Visit https://www.selenium.dev/documentation/test_practices/encouraged/fresh_browser_per_test/ and compare to https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/firefox/FirefoxDriver.html

Or copy this code snipped in your IDE:

FirefoxProfile profile = new FirefoxProfile(new File("pathToFirefoxProfile"));
WebDriver driver = new FirefoxDriver(profile);
```

### Relevant log output

```shell
IDE is telling:
"Cannot resolve constructor 'FirefoxDriver(org.openqa.selenium.firefox.FirefoxProfile)'"
```

### Operating System

Windows 10

### Selenium version

Java 4.1.1

### What are the browser(s) and version(s) where you see this issue?

does not matter

### What are the browser driver(s) and version(s) where you see this issue?

does not matter

### Are you using Selenium Grid?

No

### PR DESCRIPTION
**Pull request by jags14385**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.en.md [hunk 1]
```diff
@@ -13,8 +13,7 @@ Ideally, spin up a new virtual machine for each test.
 If spinning up a new virtual machine is not practical,
 at least start a new WebDriver for each test.
 For Firefox, start a WebDriver with your known profile.
-
+FirefoxDriver will start with a clean known state, by default
```
Comment (diemol) [inline website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.en.md]:
```suggestion
Most browser drivers like GeckoDriver and ChromeDriver  will start with a clean known state with a new user profile, by default.
```

File: website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.en.md [hunk 2]
```diff
@@ -13,8 +13,7 @@ Ideally, spin up a new virtual machine for each test.
...
 ```java
-FirefoxProfile profile = new FirefoxProfile(new File("pathToFirefoxProfile"));
-WebDriver driver = new FirefoxDriver(profile);
+WebDriver driver = new FirefoxDriver();
```
Comment (diemol) [inline website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.en.md:18]: And please remove the line `For Firefox, start a WebDriver with your known profile.`

**Review by diemol — CHANGES_REQUESTED** [review]
Can you please add these changes to the other `fresh_browser_per_test.*.md` files?

File: website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.pt-br.md [hunk 3]
```diff
@@ -12,9 +12,8 @@ Comece cada teste a partir de um estado limpo conhecido.
 Idealmente, ligue uma nova máquina virtual para cada teste.
 Se ligar uma nova máquina virtual não for prático,
 pelo menos inicie um novo WebDriver para cada teste.
-Para Firefox, inicie um WebDriver com seu perfil conhecido.
```
Comment (jags14385) [inline website_and_docs/content/documentation/test_practices/encouraged/fresh_browser_per_test.pt-br.md:15]: For the removal of this line, I used google translate to confirm

**Review by diemol — APPROVED** [review]
Thank you, @jags14385!

**Pull request merged**